README.md

```diff
@@ -1,13 +1,28 @@
-# Diy Json Serializer Lab
+# DIY JSON Serializer Lab
 
 ## Objectives
 
   1. Write a JSON Serializer
-  2. Use that JSON to render information
+  2. Use that JSON to render information without page reload
 
-## Outline
+## Introduction
 
-  * Take the solution from previous lab (The products one)
-  * Goal here is to replace the show page of products, and then to include a new link called "next product" that loads the data for the next product without page refresh
-  * Have them create a new route called like `/products/:id/data`. Write their own serializer class
+We're going to use our product/order system and enhance it so that
+customers can browse the product catalog without refreshing the page.
 
+You'll be modifying and building on the code from the previous lab,
+which is provided.
+
+## Instructions
+
+1. Create a `ProductSerializer` that serializes all of a product's
+   attributes (`name`, `price`, `inventory` and `description`) to JSON.
+2. Create a route and action for `/products/:id/data` that returns a
+   JSON-serialized `product`.
+3. Create the product `show` page and route. Add a link to the products `show` page called "Next Product" that
+   will use AJAX to load the data for the next product without refreshing the page.
+4. Make sure tests pass, including already existing ones!
+
+**Note:** The test suite makes use of `selenium-webdriver` to test the
```

Comment:
I like how this lab is structured, and I think it does a great job of letting students know what they need to do and then setting them loose.